org.opentosca.yamlconverter.main/src/main/java/org/opentosca/yamlconverter/main/UI/ConsoleUI.java

```diff
@@ -112,13 +113,23 @@ public static void main(String[] args) {
 			final String xsdfilename = promptString("\nIf you want to save this XSD, enter a filename, else just hit ENTER...");
 			if (xsdfilename != null && !xsdfilename.isEmpty()) {
 				try {
-					fileutil.saveStringAsFile(xsdfilename, xsd);
+					FileUtil.saveStringAsFile(xsdfilename, xsd);
 				} catch (final IOException e) {
 					System.out.println("ERROR: File has not been saved, because of an IOException. Muh..");
 				}
 			}
 		}
 
+		final String tempfolder = "tmp_" + System.currentTimeMillis();
+		final String csarfilename = promptString("\nIf you want to save the results as a CSAR-File, enter a filename, else just hit ENTER...");
```

Comment:
It would be cool if there is a check if the filename ends with ".csar", otherwise add it automatically. But it's just a nice-to-have ;)